templates/kapeta/resource-type-cloud-bucket/client.ts.hbs

```diff
@@ -0,0 +1,61 @@
+//#FILENAME:src/.generated/clients/{{pascalCase data.metadata.name}}Client.ts:write-always
+//
+// GENERATED SOURCE - DO NOT EDIT
+//
+
+import type { Readable } from 'stream';
+import * as minio from 'minio';
+import { ConfigProvider, ResourceInfo } from "@kapeta/sdk-config";
+
+export const RESOURCE_TYPE = '{{fullName data.kind}}';
+export const PORT_TYPE = 'http';
+export const RESOURCE_NAME = '{{data.metadata.name}}';
+export const DEFAULT_BUCKET = '{{kebabCase data.metadata.name}}';
+
+export async function create{{pascalCase data.metadata.name}}Client(config: ConfigProvider): Promise<{{pascalCase data.metadata.name}}Client> {
+  const resourceInfo = await config.getResourceInfo(RESOURCE_TYPE, PORT_TYPE, RESOURCE_NAME);
+  if (!resourceInfo) {
+    throw new Error(`Could not find resource info for ${RESOURCE_TYPE}#${PORT_TYPE} ${RESOURCE_NAME}`);
+  }
+  return new {{pascalCase data.metadata.name}}Client(resourceInfo);
```

Comment:
We usually perform a connection test before returning. Is that possible ?

Reply:
Ah, yea I can add that

Reply:
Added bucketExists check as a "ping".